enterprise_access/apps/content_assignments/models.py

```diff
@@ -0,0 +1,101 @@
+"""
+Models for content_assignments
+"""
+from django.db import models
+from django_extensions.db.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+from enterprise_access.apps.subsidy_access_policy.models import SubsidyAccessPolicy
+
+from .constants import LearnerContentAssignmentStateChoices
+
+
+class AssignmentPolicy(TimeStampedModel):
```

Comment:
Nit: We've been using a convention of throwing a `uuid` primary key on most of our new model definitions for a while, could you throw that on these two models instead of the auto big int?

Reply:
ah, of course